docs/dasharo-tools-suite/documentation.md

```diff
@@ -218,26 +226,37 @@ You could confirm that by repeating steps 3 - 5.
 
 ## Features
 
-This section describes functionality of Dasharo Tools Suite. These are:
+This section describes the functionality of the Dasharo Tools Suite. These are:
 
-* Dasharo zero-touch initial deployment
-* HCL Report,
-* Firmware update,
-* EC Transition,
-* EC update.
+* [Dasharo zero-touch initial deployment](#dasharo-zero-touch-initial-deployment),
```

Comment:
@TomaszAIR I think that in the near future, we should think about splitting the documentation into smaller files. Currently, in one file we already have described: compatible platforms, DTS building, DTS prerequisites, and all DTS features.

Reply:
@sulewskiprzemyslaw any ideas about documentation changes are welcomed here https://github.com/Dasharo/dasharo-issues/issues/337

Reply:
@TomaszAIR OK, so I will close this thread and add the appropriate entry in the dedicated issue.